samples/boards/nordic/system_off/src/main.c

```diff
@@ -99,6 +102,11 @@ int main(void)
 	}
 
 	if (do_poweroff) {
+#if CONFIG_SOC_NRF54H20_CPUAPP
+		/* Local RAM0 (TCM) is currently not used so retention can be disabled. */
+		nrf_memconf_ramblock_ret_mask_enable_set(NRF_MEMCONF, 0, RAMBLOCK_RET_MASK, false);
+		nrf_memconf_ramblock_ret_mask_enable_set(NRF_MEMCONF, 1, RAMBLOCK_RET_MASK, false);
+#endif
```

Comment:
And this also should be applied automatically by `sys_poweroff()`, unless the ram retention driver requires to keep any of these.